src/modules/table.marko

```diff
@@ -2253,6 +2253,468 @@ import tableLandscapePic from "/src/routes/static/img/tb-landscape-pic.jpg";
+                                            SKU : A43BTR5678  •  Quantity : 1
+                                        </div>
+                                    </div>
+                                </div>
+                            </td>
+                            <td class="table-cell">
+                                <span class="signal signal--trustworthy">Restocking</span>
+                            </td>
+                            <td class="table-cell table-cell--numeric">$519.99</td>
+                            <td class="table-cell table-cell--numeric">205</td>
+                            <td class="table-cell">
+                                <div class="table-cell__data">
+                                    <a>00-10542-89507</a>
+                                </div>
+                                <div class="table-cell__data table-cell__data--secondary">100</div>
+                            </td>
+                            <td class="table-cell table-cell--numeric">265</td>
+                            <td class="table-cell table-cell--numeric">$29.99</td>
+                            <td class="table-cell">FREE</td>
+                            <td class="table-cell">4/11 - 4/15</td>
+                        </tr>
+                    </tbody>
+                </table>
+                <div class="progress-bar-expressive" tabindex="0" aria-live="off">
+                    <div
+                        role="progressbar"
+                        aria-label="Loading..."
+                        class="progress-bar-expressive__progress"
+                    >
+                        <div class="progress-bar-expressive__lines">
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                            <div class="progress-bar-expressive__line"></div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <highlight-code type="html">
+<div class="table table--loading-state" role="group" aria-label="Video games for sale" tabindex="0" inert>
```

Comment:
Should table be focusable while it is loading with tabindex=0? Since its not interactive and its also not scrollable while its being loaded I would assume we should change the tabindex to -1 while its loading.

Reply:
All done!